src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,7 +9,7 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric and no whitespaces is allowed";
```

Comment:
Tag names should be alphanumeric and no whitespaces are allowed

Reply:
Edited, thanks for pointing out